docs/privacy/reviews/20251110133500_log_consent_guard_owner.md

```
@@ -0,0 +1,27 @@
+# Privacy Review — 20251110133500_log_consent_guard_owner.sql
+
+## Change
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)